src/EReg/Comms.cpp

```diff
@@ -1,11 +1,14 @@
-#include <ERegComms.h>
+#include "Comms.h"
 
-namespace ERegComms {
+namespace Comms {
 
-    commFunction callbackMap[5];
-    // EthernetUDP Udp;
+    commFunction callbackMap[numIDs] = {};
     char packetBuffer[sizeof(Packet)];
 
+    void initComms() {
+        Serial.begin(9600);
```

Comment:
is there a reason for not using faster baud rate?

Reply:
No reason, would you like 115200?

Reply:
ye